go.mod

```diff
@@ -20,8 +20,8 @@ require (
 	// NOTE: urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
 	github.com/urfave/cli v1.22.1
 	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/net v0.8.0
-	golang.org/x/sys v0.6.0
+	golang.org/x/net v0.22.0
```

Comment:
There is 0.24.0 out, this version seems to have a CVE still (https://pkg.go.dev/golang.org/x/net?tab=versions). Can we update to 0.24.0?